app.js

```diff
@@ -0,0 +1,96 @@
+// question 1
+
+// for a light switch statement you would use a if else statemet
+var light = 'bright'
+if(switchup === on){
+	return bright
+} else(swithDown === off){
+	return dark
+}
+
```

Comment:
use boolean! 
numerous other issues -- if else doesn't need return, and isn't a data structure, and the comparisons being done are of variables that haven't been declared/assigned